packages/extensions-sdk/src/cli/commands/build.ts

```diff
@@ -75,13 +82,47 @@ export default async function build(options: BuildOptions): Promise<void> {
 	const rollupOptions = getRollupOptions(type, language, input, config.plugins);
 	const rollupOutputOptions = getRollupOutputOptions(type, output);
 
-	const bundle = await rollup(rollupOptions);
+	if (options.watch) {
+		const watcher = rollupWatch({
+			...rollupOptions,
+			output: rollupOutputOptions,
+		});
+
+		watcher.on('event', async (event) => {
+			switch (event.code) {
+				case 'ERROR': {
+					spinner.fail(chalk.bold('Failed'));
+					handleRollupError(event.error);
+					spinner.start(chalk.bold('Watching files for changes...'));
+					break;
+				}
+				case 'BUNDLE_END':
+					await event.result.write(rollupOutputOptions);
```

Comment:
This line can be removed as `event.result.write()` is only meant for generating additional bundles.